iOS Client/TabController/MBTabBarController.swift

```diff
@@ -9,21 +9,62 @@
 import ReSwift
 import UIKit
 
-class MBTabBarController: UITabBarController {
+class MBTabBarController: UITabBarController, StoreSubscriber {
+    
+    var playPauseView: UIView!
+    var playPauseButton: UIButton!
+    var playerState: PlayerState = .initialized
+    
     static func instantiateFromStoryboard() -> MBTabBarController {
         // swiftlint:disable force_cast
         return UIStoryboard(name: "Main", bundle: nil).instantiateViewController(withIdentifier: "BaseTabController") as! MBTabBarController
         // swiftlint:enable force_cast
     }
+    func configurePlayPauseView() {
+        let tabBar = self.tabBar
+        playPauseView = UIView()
+        playPauseView.translatesAutoresizingMaskIntoConstraints = false
+        playPauseView.backgroundColor = UIColor.gray
+        view.addSubview(playPauseView)
+        playPauseView.heightAnchor.constraint(equalToConstant: 40).isActive = true
+        playPauseView.leadingAnchor.constraint(equalTo: view.leadingAnchor).isActive = true
+        playPauseView.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
+        playPauseView.bottomAnchor.constraint(equalTo: tabBar.topAnchor).isActive = true
+        playPauseView.isHidden = true
+        
+        playPauseButton = UIButton()
+        playPauseButton.setTitle("Play", for: .normal)
+        playPauseButton.addTarget(self, action: #selector(togglePlayPause(_:)), for: .touchUpInside)
+        playPauseButton.translatesAutoresizingMaskIntoConstraints = false
+        playPauseView.addSubview(playPauseButton)
+        playPauseButton.centerXAnchor.constraint(equalTo: playPauseView.centerXAnchor).isActive = true
+        playPauseButton.centerYAnchor.constraint(equalTo: playPauseView.centerYAnchor).isActive = true
+    }
+    
+    @objc func togglePlayPause(_ sender: UIButton) {
+        let action: Action
+        if playerState == .paused || playerState == .initialized {
+            action = ResumePodcast()
+        } else {
+            playPauseButton.setTitle("Play", for: .normal)
```

Comment:
shouldn't this happen in newState once the state is updated?

Reply:
Yeah I think I change this in my next branch but I can move it here too